Analysis/include/TriggerResults.h

```diff
@@ -50,9 +50,11 @@ class TriggerDescriptorCollection {
         LegType type;
         double pt;
         boost::optional<double> eta;
+        bool applyL1match = false;
```

Comment:
ok, if you added it as a nonoptional argument to the constructor, don't need to initialize here.